msteams-platform/concepts/build-and-test/analyze-your-apps-usage-in-developer-portal.md

```diff
@@ -46,57 +49,189 @@ After you have selected your desired filters, you can explore the following indi
+
+* Daily R30 data for the last 75 days.
+* End of month R30 data for up to 12 months.
+
+   You can select each of these metric names to see trends over time.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-app-dashboard-view.png" alt-text="Screenshots shows you app capability usage of your published app in Developer Portal."lightbox="../../assets/images/tdp/tdp-app-dashboard-view.png":::
+
+ ### Host product
+
+   The host product page shows you the following app usage analytics:
+
+   * [Usage by host product](#usage-by-host-product)
+   * [Usage by operating system](#usage-by-operating-system)
+   * [Usage by platform](#usage-by-platform)
+
+   The Key Performance Indicator (KPI) cards display metrics such as the host with the most active users and the operating system (OS) with the most active users, based on monthly, weekly, and daily data, in a metric card format.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-host-product.png" alt-text="Screenshots shows you app usage in different host products of your published app in Developer Portal.":::
+
+ #### Usage by host product
+
+   The **Usage by host product** chart shows your app's active usage across various host products, including as **Teams**, **Outlook**, and the **Microsoft 365 app** of monthly, weekly, and daily data.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-host-usage.png" alt-text="Screenshots shows you app usage in different host product of your published app in Developer Portal.":::
+
+ #### Usage by platform
+
+   The **Usage by platform** chart shows your app's active usage across various platforms, such as **Desktop**, **Mobile**, and **Web** of monthly, weekly, and daily data.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-usage-platform.png" alt-text="Screenshots shows you app usage in different platform of your published app in Developer Portal.":::
+
+   > [!NOTE]
+   > The `Preview` tag indicates that the metric **Usage by host product** is in its initial phase and might undergo modifications as we collect feedback and continue to improve the experience.
+
+ #### Usage by operating system
+
+   The **Usage by operating system** chart shows your app's active usage across various operating systems, such as **Windows**, **Mac**, **iOS**, and **Android** of monthly, weekly, and daily data.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-operating-system.png" alt-text="Screenshots shows you app usage in different OS of your published app in Developer Portal.":::
+
+ ### App capability
+
+   The app capability page shows you the usage of individual app capabilities that you've configured within your app.
+
+  This page summarizes the overall usage of the following app capabilities in different host products.
+
+  * Bot
+  * Tab
+  * Message extension
+  * Personal app
+  * Connector
+
```

Comment:
We should explain Outlook Message Extensions and Outlook Add-ins are not included. And check if we personal tab.

Reply:
Added personal tab as suggested. 

I've listed only the available capabilities metrics.